src/main/scala/notifications/Email.scala

```diff
@@ -0,0 +1,62 @@
+package notifications
+
+import java.util.{Date, Properties}
+
+import com.typesafe.config._
+import akka.actor.Actor
+import javax.mail._
+import javax.mail.internet.{InternetAddress, MimeMessage}
+import notifications.Errors.{AuthorizationError, TargetError, InternalError}
+
+class Email extends Actor {
+  val conf = ConfigFactory.load()
+  val login = conf.getString("email.login")
```

Comment:
It would be better to have EmailConfig class/object and pass it to Email constructor, then just access all members with config.login etc

Reply:
done